src/app/Article/Comment.jsx

```diff
@@ -8,10 +22,19 @@ export function Comment({ comment }) {
         </p>
       </div>
       <p className="comment-body">{comment.body}</p>
+      {user === comment.author && (
+        <img
```

Comment:
You might want this action to be disabled whilst the delete function is running, there are multiple ways to do this.